packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -541,17 +541,5 @@ export async function installScriptsAndDependencies(options: {
 }
 function getWinAppSDKPackages(nugetVersion: string): NugetPackage[] {
```

Comment:
would it be possible to also move the NETCore.UniversalWindowsPlatform import from `getUwpCsPackages()` out to a props file and get rid of these function calls altogether?

Reply:
Yeah, I wasn't sure how far to take moving things. Because I *could* go for a CoreApp-style approach and move all package versions for all templates.

For WinAppSDK, with the SDK-style projects, I'm all for that approach.

For the UWP C++ and C#, I wasn't so sure.

Reply:
I think I will leave this PR as is, and open a new bug to gauge if we want to go further.

Reply:
Opened bug here: #10829